dev/release/verify-release-candidate.sh

```diff
@@ -499,6 +499,36 @@ maybe_setup_conda() {
   fi
 }
 
+install_maven() {
+  MAVEN_VERSION=3.8.7
+  SYSTEM_MAVEN_VERSION=$(mvn -v | head -n 1 | awk '{print $3}')
+
+  if [[ "$MAVEN_VERSION" == "$SYSTEM_MAVEN_VERSION" ]]; then
+      echo "Skip Install - versions are the same"
+  else
+      # the following logic only verifies released versions
+      older_version=$(printf '%s\n%s\n' "$SYSTEM_MAVEN_VERSION" "$MAVEN_VERSION" | sort --version-sort | head -n1)
+      if [[ "$older_version" == "$SYSTEM_MAVEN_VERSION" && "$SYSTEM_MAVEN_VERSION" != "$MAVEN_VERSION" ]]; then
```

Comment:
Could you remove redundant condition?

```suggestion
      if [[ "$older_version" == "$SYSTEM_MAVEN_VERSION" ]]; then
```